### ISSUE TITLE
Batch deleting products throws exceptions

### ISSUE THREAD

**Issue by gaazkam** [XiaoFaye/WooCommerce.NET#575]
- Wordpress version, WooCommerce version and WooCommerce.NET version

Wordpress 5.6.2, Woo 5.0.0 (a clean XAMPP / Bitnami installation set up just to test this issue)

WooCommerce.NET version 0.8.2

- Steps to replicate the issue

Add a product. Then delete it using WooCommerce.NET, for example with the following code (assuming the id of the product to be deleted is 14):

```csharp
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WooCommerceNET;
using WooCommerceNET.WooCommerce.v3;

namespace ConsoleApp3
{
    class Program
    {
        static async Task Main(string[] args)
        {
            var wc = new WCObject(new RestAPI
            (
                "http://localhost/wordpress/wp-json/wc/v3/",
                "redacted",
                "redacted",
                requestFilter: request =>
                {
                    request.UserAgent = "TestException";
                    request.Accept = "application/json";
                }
            ));

            await wc.Product.UpdateRange(new ProductBatch { delete = new List<int> { 14 } });

            Console.WriteLine("done");

            Console.ReadLine();
        }
    }
}
```

(Target framework: .NET Framework 4.8)

The `wc.Product.UpdateRange` line throws an exception. The product, however, seems to be deleted sucesfully.

- Details of the error message if there is any

```
System.Runtime.Serialization.SerializationException
  HResult=0x8013150C
  Message=There was an error deserializing the object of type WooCommerceNET.Base.BatchObject`1[[WooCommerceNET.WooCommerce.v3.Product, WooCommerce.NET, Version=0.8.2.0, Culture=neutral, PublicKeyToken=ff41d0b6c937438d]]. The value '' cannot be parsed as the type 'Int32'.
  Source=WooCommerce.NET
  StackTrace:
   at WooCommerceNET.RestAPI.DeserializeJSon[T](String jsonString)
   at WooCommerceNET.Base.WCItem`1.<UpdateRange>d__16.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at ConsoleApp3.Program.<Main>d__0.MoveNext() in C:\Users\WTF\ConsoleApp3\ConsoleApp3\Program.cs:line 27

  This exception was originally thrown at this call stack:
    [External Code]

Inner Exception 1:
XmlException: The value '' cannot be parsed as the type 'Int32'.

Inner Exception 2:
FormatException: Input string was not in a correct format.
```

**Comment by XiaoFaye** [comment]
This is a duplicate of #523 , will be fixed in version 0.8.3 very soon.

**Comment by gaazkam** [comment]
Thank you.

And sorry for posting a dupe.
